orttraining/orttraining/training_ops/cpu/loss/softmax_cross_entropy_loss.cc

```diff
@@ -386,6 +386,16 @@ Status SoftmaxCrossEntropyLossGrad<T1, T2>::Compute(OpKernelContext* context) co
     d_logit->Reshape(new_shape);
   }
 
+  // Bias.
```

Comment:
I am a bit surprised we did not use parallel for here. maybe for the newly added elementwise add, it's simple to a parallel loop.

Reply:
It may means no body use CPU for training, so I didn't take effort to refactor the code, and just to make it work.

Reply:
If it's critical for on-device training, we can optimize this in a new PR.